jelares/portfolio/src/main/webapp/javascript/script.js

```diff
@@ -0,0 +1,153 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Changes the text of an element to a randomly chosen string
+ */
+function addRandomText(textChoices, elementId) {
+  // Pick a random text.
+  const textChoice =
+      textChoices[Math.floor(Math.random() * textChoices.length)];
+
+  // Add it to the page.
+  const textContainer1 = document.getElementById(elementId + '1');
+  const textContainer2 = document.getElementById(elementId + '2');
+
+  if (textContainer1.style.opacity === '0' &&
+      textContainer2.style.opacity === '0') {
+    textContainer1.innerText = textChoice;
+    textContainer2.style.opacity = 0;
+    textContainer1.style.opacity = 1;
+
+  } else if (textContainer1.style.opacity === '1') {
+    textContainer2.innerText = textChoice;
+    textContainer1.style.opacity = 0;
+    textContainer2.style.opacity = 1;
+
+  } else {
+    textContainer1.innerText = textChoice;
+    textContainer2.style.opacity = 0;
+    textContainer1.style.opacity = 1;
+  }
+}
+
+// Random Greeting
```

Comment:
Consider adding to the documentation to better inform the user what this function will do. Example:
Adds a random greeting to either the greeting-container1 or greeting-container2 element.

Also applies to functions below.

Reply:
Updated